package.json

```diff
@@ -53,6 +55,7 @@
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
     "@ianvs/prettier-plugin-sort-imports": "4.1.0",
+    "@mui/icons-material": "^5.14.9",
```

Comment:
Instead of installing the icon library, I recommend copy `<svg>` and pass it to SvgIcon component instead:

```js
import SvgIcon from '@mui/joy/SvgIcon';

<SvgIcon>
  <svg>…</svg>
</SvgIcon>
```

The reason is to keep `@mui/joy` as the only dependency to give the best copy & paste DX.